### ISSUE TITLE
AWS::Include in Mappings causes errors in cfn-lint (while being correct CloudFormation template)

### ISSUE THREAD

**Issue by pbudzon** [aws-cloudformation/cfn-lint#3147]
### CloudFormation Lint Version

cfn-lint 0.86.3

### What operating system are you using?

MacOS

### Describe the bug

Cfn-lint doesn't understand `Fn::Transform` and `AWS::Include` within `Mappings` section of the template and returns the following errors:

```
E7002 Mapping Fn::Transform has invalid name.  Name has to be alphanumeric.
test.yaml:5:3

W7001 Mapping 'Fn::Transform' is defined but not used
test.yaml:5:3

E7001 Mapping Fn::Transform has invalid property at AWS::Include
test.yaml:6:5

E1011 Map Name mapname does not exist for mapname
test.yaml:16:25
```

We store mappings in a separate file so they can be reused in multiple templates that require the same set of information.

### Expected behavior

cfn-lint should either 
1) understand `AWS::Include` 
OR
2) show a warning that it doesn't evaluate Includes  and **not** show errors on the related resources

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Description: Example

Mappings:
  Fn::Transform:
    Name: AWS::Include
    Parameters:
      Location: mappings.yaml

Resources:
  ServiceDiscoveryNamespace:
    Type: AWS::ServiceDiscovery::PrivateDnsNamespace
    Properties:
      Description: !Ref "AWS::StackName"
      Name: example.com
      Vpc: !FindInMap [ mapname, !Ref "AWS::Region", VPC ]
```

mappings.yaml:
```yaml
mapname:
  eu-west-1:
    VPC: vpc-abc
```

**Comment by PatMyron** [comment]
https://github.com/aws-cloudformation/cfn-lint/issues/476
